ptx/sec_related_rates.ptx

```diff
@@ -1116,15 +1116,15 @@
           </pg-code>
           <introduction>
             <p>
-              An F-22 aircraft is flying at
+              An F-22 aircraft is flying at <m><var name="$spd"/>\,\text{mph}</m>
```

Comment:
The webwork variable `$spd` is already a number with units. This will result in a display like 530 mph mph.